### ISSUE TITLE
Does this support Postgres array type?

### ISSUE THREAD

**Issue by bfkelsey** [orangy/squash#4]
Do you have support or any plans to support Postgres array? 
https://www.postgresql.org/docs/9.1/static/arrays.html

**Comment by orangy** [comment]
No, right now I try to support a uniform basic SQL functionality. Not exactly ANSI SQL, but whatever can be represented in most databases. I plan to introduce extensions for specific databases later, though you can do it yourself, if you wish.

**Comment by dudleyf** [comment]
@orangy Could you say a little more about what you were thinking as far as database-specific extensions? I've been thinking about doing just that for Postgres.

**Comment by codesplode** [comment]
@dudleyf I've been working in the squash source for a little bit now. I use MySQL, but I have found my way around quite a bit. Given how busy @orangy is with the Kotlin team, I figured I would try to help. The class you want to look at is PgDialect. It looks like @orangy already did some postgres specific work there, but it is a good place for working with new column types.

See: [PgDialect.kt](https://github.com/orangy/squash/blob/master/squash-postgres/src/org/jetbrains/squash/dialects/postgres/PgDialect.kt)

My team uses MySQL, but I have looked at the unit tests for all databases and recommend those as a good start for seeing how squash interacts. That's where I started and it helped a lot.
